docs/HowToFAQ/ERROR_ModifiesValue1.dfy

```diff
@@ -0,0 +1,5 @@
+class A {}
```

Comment:
What references this file?

Reply:
ERROR_ModifiesValue.md. For some reason ERROR_ModifiesValue1.dfy was missing and the builds of the website (though not the CI builds) were failing as a result.